model_armor/create_template.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from google.cloud.modelarmor_v1 import Template
+
+
+def create_model_armor_template(project_id: str, location: str, template_id: str) -> Template:
+    # [START modelarmor_create_template]
+
+    from google.api_core.client_options import ClientOptions
+    from google.cloud import modelarmor_v1
+    from google.cloud.modelarmor_v1 import Template, DetectionConfidenceLevel, FilterConfig, PiAndJailbreakFilterSettings, \
+        MaliciousUriFilterSettings
+
+    client = modelarmor_v1.ModelArmorClient(
+        transport="rest",
+        client_options=ClientOptions(api_endpoint=f"modelarmor.{location}.rep.googleapis.com"),
+    )
+
+    # TODO(Developer): Uncomment these variables and initialize
+    # project_id = "your-google-cloud-project-id"
+    # location = "us-central1"
+    # template_id = "template_id"
+
+    template = Template(
+        name=f"projects/{project_id}/locations/{location}/templates/{template_id}",
+        filter_config=FilterConfig(
+            pi_and_jailbreak_filter_settings=PiAndJailbreakFilterSettings(
+                filter_enforcement=PiAndJailbreakFilterSettings.PiAndJailbreakFilterEnforcement.ENABLED,
+                confidence_level=DetectionConfidenceLevel.MEDIUM_AND_ABOVE,
+            ),
+            malicious_uri_filter_settings=MaliciousUriFilterSettings(
+                filter_enforcement=MaliciousUriFilterSettings.MaliciousUriFilterEnforcement.ENABLED,
+            )
+        ),
+    )
+
+    # Initialize request arguments
+    request = modelarmor_v1.CreateTemplateRequest(
+        parent=f"projects/{project_id}/locations/{location}",
+        template_id=template_id,
+        template=template,
+    )
+
+    # Make the request
+    response = client.create_template(request=request)
+
+    # Response
+    return response
+
+
+# [END modelarmor_create_template]
```

Comment:
https://googlecloudplatform.github.io/samples-style-guide/ "Process the result" > "show the user how to interact with a returned object programmatically by printing example attributes to the console."

I would suggest changing this to have the template printed, then the `[END` region tag, then the return statement to support testing. 

I'm not sure how to interact with the template so I don't know if you can just `print(response)` or if it requires more steps, so I can't make a suggested change. 

This comment applies to all other samples that return before the `[END`.

Reply:
Also, having a `return` outside of a Python method is invalid so this sample would not be copy/pastable, even after replacing the project_id/location/template_id variables.

Reply:
Thanks for the catch! I've updated to print the response. And added the region tag before return.